### PR TITLE
Add TSK story cards from campaign

### PR DESCRIPTION
**Pull request by zzorba**
Already released in the netherlands/belgium, should be widely released on Nov 18.

### CODE REVIEW

File: pack/tsk/tskc.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    "encounter_position": 4,
+    "exile": true,
+    "faction_code": "neutral",
+    "illustrator": "Grzegorz Bobrowski",
+    "name": "Expedited Ticket",
+    "pack_code": "tskc",
+    "position": 265,
+    "quantity": 1,
+    "permanent": true,
+    "text": "Permanent.\nWhile embarking to a new destination on the map, you may exile Expedited Ticket and mark 1 <b>time</b> in your Campaign Logto travel anywhere on the map.",
```
Comment (drawntotheflame) [inline pack/tsk/tskc.json:86]:
```suggestion
    "text": "Permanent.\nWhile embarking to a new destination on the map, you may exile Expedited Ticket and mark 1 <b>time</b> in your Campaign Log to travel anywhere on the map.",
```

**Review by drawntotheflame — COMMENTED** [review]
Good job @zzorba

File: pack/tsk/tskc.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
+[
+  {
+    "code": "09762",
```
[the remark this answers is not in the document]
Comment (zzorba) [reply pack/tsk/tskc.json:3]: Correct, that's what we did for EOE so I figured we would continue here.

**Comment by zzorba** [comment]
All changes are in this can be merged now.

**Pull request merged**